docs/technical-docs/issues-workflow/psychopy.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Custom Workflows
+parent: General Issues Workflow
+---
+
+# Custom Workflows
+
+## Overview
+
+Custom Issues workflow for specific projects in the lab where the issues are used for more than just
+posting bugs and feature requests.
+
+Reference the [General Issues Workflow](./index.md) document to get a more in-depth description about the
+workflow, and view additional resources.
+
+Sections:
+
+Labels that will signal the kind of issue to make it easier for contributors to find issues that
+they can help with more easily.
+
+Templates:
+
+Template forms for the issues that will provide context to what information is needed to submit an
+issue.
```

Comment:
Is "sections" and "templates" a standard thing that will appear in a lot of docs? If so, I get this layout, if not, seems odd to have headings with one thing each.